stdlib/public/core/Set.swift

```diff
@@ -920,6 +921,9 @@ extension Set: SetAlgebra {
   @inlinable
   internal mutating func _subtract<S: Sequence>(_ other: S)
   where S.Element == Element {
+    // If self is empty we don't need to iterate over `other` because there's nothing to remove on self.
```

Comment:
Nit: 80 columns here, otherwise looks good!
```suggestion
    // If self is empty we don't need to iterate over `other` because there's
    // nothing to remove on self.
```